pkg/sources/adapter/azureservicebussource/adapter.go

```diff
@@ -180,27 +190,35 @@ func entityPath(entityID *v1alpha1.AzureResourceID) string {
 	}
 }
 
-// namespaceFromEnvironment mimics the behaviour of eventhub.NewHubFromEnvironment
```

Comment:
I see this comment was removed, but the behaviour still mimics `eventhub.NewHubFromEnvironment`. Keeping this specific part might help explain the design to readers who are unfamiliar with the code, since they know they should refer to the docs of the `eventhub` module to get the context of why we did it that way.